### PR TITLE
update zh-cn

### PR DESCRIPTION
**Pull request by BluesDawn576**

### CODE REVIEW

**Comment by micwoj92** [comment]
@chenyiping1995 do you approve these?

File: Langs/0804.lang [hunk 1]
```diff
@@ -20,24 +30,25 @@ RAM Installed=内存大小
...
 64 Bit OS=64 位操作系统
 32 Bit CPU=32 位 CPU
 32 Bit OS=32 位操作系统
 Unable to Determine=无法确定
 Unable to Check List=无法验证
 Error Accessing List=读取兼容列表时出错
-Not Currently Listed as Compatible=未在兼容列表中列出
-Listed as Compatible=在兼容列表中
+Not Currently Listed as Compatible=与系统要求不兼容
+Listed as Compatible=符合系统要求
```
Comment (chenyiping1995) [inline Langs/0804.lang:42]: I dont think translations for line 41 and 42 convey the original meaning.

*Not on the compatible list* does not necessarily mean *incompatible* or *not meeting system requirement*. It is the user's judgment after all.

You can't plain tell the users that their machine does not meet the system requirements, just becauae their CPU is *not listed as compatible*.

I'd suggest keeping the original translation for line 41 to 42.

File: Langs/0804.lang [hunk 2]
```diff
@@ -53,7 +64,22 @@ You're running the latest build!=您当前使用的 WhyNotWin11 是最新版本
...
+Boot=指主板加载 Windows 的方式。如果结果为“Legacy”，则可以在较新的主板中找到 BIOS/UEFI 设置以进行调整。具体请参阅您的主板使用手册。
+CPU Name=指在计算机中安装的 CPU。与系统的兼容性可能会发生变化。如果您使用的是台式电脑，则可以更换 CPU；笔记本电脑一般无法更换。
+CPU Cores=指 CPU 可以同时处理的任务量。如果您使用的是台式电脑，则可以更换 CPU；笔记本电脑一般无法更换。
+CPU Speed=指 CPU 处理任务的速度。如果您使用的是台式电脑，则可以更换 CPU；笔记本电脑一般无法更换。
+DirectX=指显卡支持的 DirectX DDI/功能层版本。与 DirectX 的软件版本不同，“DirectX 12 API”的显卡可能无法通过此检查。
+Disk Type=指磁盘储存数据的方式。如果分区形式不是 GPT，则可以使用 Microsoft 提供的 MBR2GPT 工具进行修复。
+RAM=指计算机中安装的内存大小。可在台式电脑和高端笔记本电脑升级。
+Secure Boot=指主板加载 Windows 的方式。如果未检测到安全启动，则可以在较新的主板中找到 BIOS/UEFI 设置以进行调整。具体请参阅您的主板使用手册。
+Storage=指磁盘中数据可使用的空间量。有些高端笔记本电脑可以更换或升级。
+TPM=指 Windows 使用的安全模块。所有新款 AMD CPU 自带此模块，部分新款 Intel CPU 自带。请检查主板中的 BIOS/UEFI 设置，具体请参阅您的主板使用手册。
```
Comment (chenyiping1995) [inline Langs/0804.lang:85]: Haven't got the chance to read the original text to the descriptions section yet. I will do that in the coming morning.

File: Langs/0804.lang [hunk 3]
```diff
@@ -53,7 +64,22 @@ You're running the latest build!=您当前使用的 WhyNotWin11 是最新版本
...
+Architecture=指 CPU 和操作系统可以一次处理的数据量。如果操作系统为 32 位，则需要格式化系统盘之后才能安装 Windows 11。如果 CPU 为 32 位，则需要更换 CPU。
+Boot=指主板加载 Windows 的方式。如果结果为“Legacy”，则可以在较新的主板中找到 BIOS/UEFI 设置以进行调整。具体请参阅您的主板使用手册。
+CPU Name=指在计算机中安装的 CPU。与系统的兼容性可能会发生变化。如果您使用的是台式电脑，则可以更换 CPU；笔记本电脑一般无法更换。
+CPU Cores=指 CPU 可以同时处理的任务量。如果您使用的是台式电脑，则可以更换 CPU；笔记本电脑一般无法更换。
+CPU Speed=指 CPU 处理任务的速度。如果您使用的是台式电脑，则可以更换 CPU；笔记本电脑一般无法更换。
+DirectX=指显卡支持的 DirectX DDI/功能层版本。与 DirectX 的软件版本不同，“DirectX 12 API”的显卡可能无法通过此检查。
+Disk Type=指磁盘储存数据的方式。如果分区形式不是 GPT，则可以使用 Microsoft 提供的 MBR2GPT 工具进行修复。
+RAM=指计算机中安装的内存大小。可在台式电脑和高端笔记本电脑升级。
+Secure Boot=指主板加载 Windows 的方式。如果未检测到安全启动，则可以在较新的主板中找到 BIOS/UEFI 设置以进行调整。具体请参阅您的主板使用手册。
+Storage=指磁盘中数据可使用的空间量。有些高端笔记本电脑可以更换或升级。
```
Comment (chenyiping1995) [inline Langs/0804.lang:84]: 指磁盘中可用于存储数据的容量。高端笔记本电脑可以更换或升级。

File: Langs/0804.lang (hunk 2 above)
Comment (chenyiping1995) [inline Langs/0804.lang:85]: 指 Windows 使用的安全模块。所有现代 AMD 处理器和一部分现代 Intel 处理器自带此模块。请检查主板中的 BIOS/UEFI 设置。具体请参阅您的主板使用手册。

**Review by chenyiping1995 — CHANGES_REQUESTED** [review]
Description section has been reviewed, and a few changes have been proposed.

**Comment by BluesDawn576** [comment]
@chenyiping1995 ok

**Comment by chenyiping1995** [comment]
@micwoj92 I don't have anything else to comment. You can merge these if you want to.

**Pull request merged**